### PR TITLE
Try restarting chromedriver on timeout

### PR DESCRIPTION
**Pull request by ijjk**
Also added `HEADLESS` env var for local testing

### CODE REVIEW

File: test/lib/next-webdriver.js [hunk 1]
```diff
@@ -17,6 +22,22 @@ export default async function (appPort, pathname) {
...
       console.warn(`> Error when loading browser with url: ${url}`)
+
+      // Try restarting chromedriver max twice
+      if (lc < 2) {
+        const chromedriver = require('chromedriver')
+        console.log('Trying to restart chromedriver with random port')
+        driverPort = await getPort()
+        chromedriver.stop()
+        chromedriver.start([`--port=${driverPort}`])
+        // https://github.com/giggio/node-chromedriver/issues/117
```
Comment (Janpot) [inline test/lib/next-webdriver.js]: Looking at https://github.com/giggio/node-chromedriver/pull/177 It seems like when you call it like

```js
await chromedriver.start([`--port=${driverPort}`], true)
```

You can eliminate the `wait-port` dependency.

Comment (ijjk) [reply test/lib/next-webdriver.js]: That's true, although it looks like the timeout is fixed that way and unfortunately I think we need to allow the timeout to be higher than 10 seconds. Thanks for looking

Comment (Janpot) [reply test/lib/next-webdriver.js]: ok, I filed https://github.com/giggio/node-chromedriver/issues/193

**Comment by oliviertassinari** [comment]
@ijjk To mitigate the `github-actions` comment noise, you might be interested in having a look at the React danger solution or https://github.com/mui-org/material-ui/pull/14587.

**Review by timneutkens — APPROVED** [review]

**Pull request merged**